akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/BasicClusterExampleSpec.scala

```diff
@@ -51,6 +51,8 @@ class BasicClusterConfigSpec extends WordSpec with ScalaFutures with Eventually
 
   import BasicClusterExampleSpec._
 
+  implicit override val patienceConfig = PatienceConfig(timeout = Span(5, Seconds))
```

Comment:
Should we define a decent `PatienceConfig` in one single place so that we don't start defining arbitary timeouts all over the place? I assume that when we use some test base class we already have that in place, but might be good with a `val` in an `object` also for this kind of test.

Reply:
Will mixin TypedSpec instead